dask_sql.Dockerfile

```diff
@@ -0,0 +1,34 @@
+ARG CUDA_VER=11.2
+ARG LINUX_VER=ubuntu18.04
+
+FROM gpuci/miniconda-cuda:$CUDA_VER-devel-$LINUX_VER
+
+ARG CUDA_VER=11.2
+ARG PYTHON_VER=3.8
+ARG NUMPY_VER=1.21
+ARG RAPIDS_VER=21.08
+ARG UCX_PY_VER=0.21
+
+ADD https://raw.githubusercontent.com/dask-contrib/dask-sql/main/conda.txt /dask_sql_requirements.txt
+
+RUN conda config --set ssl_verify false
+
+RUN conda install -c gpuci gpuci-tools
+
+RUN gpuci_conda_retry install -c conda-forge mamba
+
+RUN gpuci_mamba_retry env create -n dask_sql --file /dask_sql_requirements.txt -c conda-forge
+
+RUN gpuci_mamba_retry install -y -n dask_sql -c rapidsai -c rapidsai-nightly -c nvidia -c conda-forge \
```

Comment:
Do we want to flip these? I usually install RAPIDS first and others things second to minimize conflicts.

Reply:
Maybe, I just pulled this env creation / install order from [one of the other Dockerfiles](https://github.com/rapidsai/dask-build-environment/blob/3a0dde38587aec3b341b65dba2bbff9e90fafac4/dask.Dockerfile#L21-L31) but I imagine this particular ordering shouldn't be too important

Reply:
Ok no worries